src/main/resources/templates/application/prior-authority-remove.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en" th:replace="~{layout :: layout(title=~{::title},mainContent=~{::#main-content},pageCategory=${'cases'},breadcrumbs=~{::#breadcrumbs})}" xmlns:th="http://www.thymeleaf.org">
+<head>
+  <title>LAA CCMS - Start new application</title>
+</head>
+<body>
+<div class="govuk-width-container">
+  <div id="breadcrumbs">
+    <a th:href="@{/application/proceedings-and-costs#prior-authority}" class="govuk-back-link">Cancel and return to proceedings and costs</a>
+  </div>
+  <div id="main-content">
+    <form action="#" th:action="@{/application/prior-authorities/{priorAuthorityId}/remove(priorAuthorityId=${priorAuthority.id})}" method="post">
+      <h1 class="govuk-heading-l">
+        Prior Authority Request
```

Comment:
Request?

Reply:
it was the same as the PUI, amended to 'Remove prior authority'